test/hack/checkLabels/main.go

```diff
@@ -103,32 +143,57 @@ func getMissingLabels() ([]string, error) {
 
 	// Get the list of labels with prefix "cancel_if_"
 	cancelLabels := getLabels(rulesList, "cancel_if_")
+	targetLabels, sourceLabels := getTargets(alertConf)
 
 	for _, cancelLabel := range cancelLabels {
-		var originLabel = strings.Split(cancelLabel, "cancel_if_")
-		if len(originLabel) < 2 {
-			return nil, fmt.Errorf("No origin label found in %q", cancelLabel)
+		var i = 0
+
+		for _, targetLabel := range targetLabels {
+			if cancelLabel == targetLabel {
+				i++
+			}
+		}
+
+		if i == 0 {
+			missingCancelLabels = append(missingCancelLabels, cancelLabel)
 		}
+	}
 
-		var originLabelList = getLabels(rulesList, originLabel[1])
+	for _, source := range sourceLabels {
+		var originLabelList = getLabels(rulesList, source)
 
-		// If a label corresponding to the "cancel_if_" prefixed one was not found, one can add it to the list of missing origin labels
+		// If a label corresponding to source one was not found, one can add it to the list of missing origin labels
 		if len(originLabelList) == 0 {
-			missingLabels = append(missingLabels, originLabel[1])
+			missingLabels = append(missingLabels, source)
 		}
 	}
 
-	return missingLabels, nil
+	return missingLabels, missingCancelLabels, nil
 }
 
 func main() {
-	missingLabels, err := getMissingLabels()
+	missingTargetLabels, missingCancelLabels, err := getMissingLabels()
 	if err != nil {
 		log.Fatal(err)
 	}
 
-	fmt.Println("MISSING LABELS:")
-	for _, label := range missingLabels {
+	fmt.Println("MISSING TARGET LABELS:")
```

Comment:
This message was not clear to me when I ran it locally.
What about:
```suggestion
	fmt.Println("MISSING TARGET LABELS IN ALERTMANAGER INHIBITIONS DEFINITIONS:")
```

Reply:
Added